util/files/tar.go

```diff
@@ -0,0 +1,195 @@
+package files
```

Comment:
The `util` package has so many sub-packages already. I think we should try to categorize them. Please consider moving `files` sub-package under existing `io` subpackage ( `util/io/files` )

Reply:
done